tools/cbmc/proofs/HTTP/global_state_HTTP.c

```diff
@@ -100,15 +102,15 @@ size_t IotNetworkInterfaceReceive( void * pConnection,
 }
 
 IotNetworkError_t IotNetworkInterfaceCallback( void * pConnection,
-					       IotNetworkReceiveCallback_t
-					         receiveCallback,
-					       void * pContext ) {
+                                               IotNetworkReceiveCallback_t
```

Comment:
Looks like a lot of useless whitespace changes, is your editor set to change between tabs and spaces?

Reply:
The file had mixed spaces and tabs everywhere. Mostly spaces, so I normalized everything to spaces.